### PR TITLE
Reduce compile times on Windows from 45s to 33s

### PR DESCRIPTION
**Pull request by Pombal**
Remove regex feature from env-logger
Remove zstd and bzip2 from zip, only deflate is needed Remove async usage in an attempt to remove tokio but that's only possible if we also remove the dependencies on reqwest/rustls-tls and windows-sys/mio

### CODE REVIEW

**Comment by Pombal** [comment]
@FredrikNoren Please test this again before merging, especialy the zip functionality. I may have missed use cases.

**Comment by Pombal** [comment]
Just saw an error with runtime set-default, checking

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ambient"
-version = "0.2.7"
+version = "0.3.0"
```
Comment (FredrikNoren) [inline Cargo.toml:3]: Think by semver it would be 0.2.8 (since there's no feature changes), but doesn't really matter

Comment (Pombal) [reply Cargo.toml:3]: I thought making it non-blocking could be considered a breaking change but I'll change it to 0.2.8

**Review by FredrikNoren — APPROVED** [review]
Awesome! Seems to work here. @ten3roberts can you quickly test on linux? Think you can just run:
```sh
cargo run -- runtime set-default 0.3.0-nightly-2023-10-04
```
(or some version that isn't installed) and we'll see if unziping works there too

**Comment by philpax** [comment]
Change looks good (and on @ten3roberts's suggestion I might do the same for semantic/the macro as I don't think it actually does simultaneous fetching right now), but what's the thinking behind ignoring some of the potential errors with `let _ = `?

**Comment by FredrikNoren** [comment]
Oh good catch, yeah those needs to be fixed. @Pombal Writing `let _ =` will ignore errors; you should use `?` instead. You can read about Result and how it works here: https://doc.rust-lang.org/rust-by-example/error/result.html

**Review by philpax — APPROVED** [review]

**Comment by philpax** [comment]
Just need to test on Linux and I think we're good to go.

**Comment by ten3roberts** [comment]
I'll fire up my linux tower and give it a whirl

**Comment by ten3roberts** [comment]
image

Seems to work on linux too

**Review by ten3roberts — APPROVED** [review]
As suspected, there were no co-dependent futures, so s'all good.

Thank you for carrying this idea through :heart:

**Pull request merged**